scripts/api/entity/shiptemplatebasedobject.lua

```diff
@@ -26,13 +26,27 @@ function Entity:setTemplate(template_name)
             comp[key] = value
         end
     end
-    if template.__type == "station" then
+    if not comp.ai_controller and not comp.player_control then
+        -- no AI or player control, this is a station
```

Comment:
This doesn't have to be true, a station could have lasers/missiles and then it also needs AI. It's not something implemented yet, but this would rule that out completely.